### DISCUSSION TITLE
Game pixelated in fullscreen

### DISCUSSION THREAD

**Discussion by Yuguiboy** [ppy/osu#16146] [Q&A]
I've just installed the last release of lazer and my game is very pixelated in fullscreen like if it weren't in the right resolution. However, the resolution is sets on 1920x1080 and that's my monitor's resolution. Am I missing something?
Screenshots I tried to take didn't show the issue, so I took photos.

![IMG_0464](https://user-images.githubusercontent.com/96346521/146647768-ecefb95d-22ee-4173-ac6d-571cd399ef2b.jpg)

![IMG_0463](https://user-images.githubusercontent.com/96346521/146647771-842e8933-c6a3-4972-ade8-ba430010afd1.jpg)

**Comment by bdach** [comment]
Can you post game logs? They're at:

* `%AppData%/osu/logs` (on Windows),
* `~/.local/share/osu/logs` (on Linux and macOS),
* `Android/Data/sh.ppy.osulazer/logs` (on Android).

Also, if on windows, attach DirectX diagnostic info by performing the following steps:

1. Press <kbd>Windows</kbd>+<kbd>R</kbd>, type `dxdiag` in the appearing window and press <kbd>Enter</kbd>.
2. You could be prompted if you'd want Windows to check if the device drivers are signed - there should be no harm in agreeing to that.
3. In the *DirectX Diagnostic Tool* window that will appear, click *Save All Information...* and choose a file to save to.
4. Attach the saved file to this issue.

I'm thinking that for some reason your graphics card may be selecting wrong mipmap sizes which would indeed result in the game looking pixelly.

**Reply by Yuguiboy** [reply]
> Can you post game logs? They're at:
> 
> * `%AppData%/osu/logs` (on Windows),
> * `~/.local/share/osu/logs` (on Linux and macOS),
> * `Android/Data/sh.ppy.osulazer/logs` (on Android).
> 
> Also, if on windows, attach DirectX diagnostic info by performing the following steps:
> 
> 1. Press Windows+R, type `dxdiag` in the appearing window and press Enter.
> 2. You could be prompted if you'd want Windows to check if the device drivers are signed - there should be no harm in agreeing to that.
> 3. In the _DirectX Diagnostic Tool_ window that will appear, click _Save All Information..._ and choose a file to save to.
> 4. Attach the saved file to this issue.
> 
> I'm thinking that for some reason your graphics card may be selecting wrong mipmap sizes which would indeed result in the game looking pixelly.

Here are all the files :
[logs.zip](https://github.com/ppy/osu/files/7739877/logs.zip)
[DxDiag.txt](https://github.com/ppy/osu/files/7739880/DxDiag.txt)

**Reply by bdach** [reply]
unfortunately i can't really see anything notable in the logs above. definitely would never suspect an RTX card to have this sort of problem...

a few more steps / sanity checks, if you don't mind:

* drivers look a touch out of date (last updated October) - could you try upgrading and seeing if that does anything?
* do you perhaps have some custom settings in NVIDIA experience? like changing LOD bias or something similar?
* also this is going to be a long shot but do you have shadowplay on, and can you try switching it off if so?

**Reply by Yuguiboy** [reply]
Thanks for your help.
I tried and checked everything you said, but unfortunately nothing worked. 
Also, I'm not really sure about that, but I haven't tried lazer since my old GPU wich was a GTX 1060. Maybe the issue is actually from the RTX card?

**Reply by bdach** [reply]
difficult to say for sure. it may be the card but i'm essentially guessing in the dark as i've never seen a report such as this.

not sure how to proceed further. to confirm my understanding, is lazer the only game with exclusive fullscreen that you have this issue on?

**Reply by peppy** [reply]
Are you using a custom DPI setting? (aka "Scaling" in windows settings)
